pacer.js

```diff
@@ -113,11 +113,17 @@ let PACER = {
     for (let url of urls) {
       let hostname = getHostname(url);
       // JS is trash. It lacks a way of getting the TLD, so we use endsWith.
+      //
+      // xxx aren't we only invoked when we match the "matches" and
+      // "include_globs" in manifest.json, which limits us to
+      // *://*.uscourts.gov/* ?
```

Comment:
The comment on line 112  alludes to this, but this function also parses referers, which can have other domains in them. The other way to do this would be to make the match more specific than "find me some numbers after a question mark at the end of a string." But I think it's OK as is?

Reply:
Right. clarified.